vic/drivers/cesm/src/cesm_put_data.c

```diff
@@ -147,9 +148,14 @@ vic_cesm_put_data()
 
                 // albedo: direct , visible
                 // CESM units: unitless
-                albedo = AreaFactor *
-                         (atmos->shortwave[NR] /
-                          (atmos->shortwave[NR] - energy.NetShortAtmos));
+                if (atmos->shortwave[NR] > 0.) {
+                    albedo = AreaFactor *
+                             (atmos->shortwave[NR] /
+                              (atmos->shortwave[NR] - energy.NetShortAtmos));
+                }
+                else {
+                    albedo = 0.;
```

Comment:
Does `albedo = 0.` create problems for the other component models in CESM (not sure what they expect from albedo at night)? What happens during the first time step of the day (since this is technically the albedo for the next time step I think)? Check with Tony. I would go ahead and accept this, but open an issue to check that this is OK.